src/pre_defined_classes/mathematical_program/hard/q1/MainTest.java

```diff
@@ -0,0 +1,33 @@
+package pre_defined_classes.mathematical_program.hard.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons (with distractors)
```

Comment:
is this the way we decided on for these? I feel like I see it differently, but I'll leave the final call on that up to @aadiraju

Reply:
I think we decided not to have brackets so this would be `// Parsons with Distractors` instead.